sources/packages/api/src/database/entities/student-appeal.model.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToMany,
+  PrimaryGeneratedColumn,
+  RelationId,
+} from "typeorm";
+import { Application } from ".";
+import { ColumnNames, TableNames } from "../constant";
+import { RecordDataModel } from "./record.model";
+import { StudentAppealRequest } from "./student-appeal-requests.model";
+
+/**
+ * Represents as set of appeals requested by a student, for instance, to have his income
+ * or dependents data changed on his Student Application after it was completed.
+ */
+@Entity({ name: TableNames.StudentAppeal })
```

Comment:
shouldn't the table name be plural?

Reply:
We do have both but I agree to keep the constants more aligned with the real table name.
![image](https://user-images.githubusercontent.com/61259237/157290052-be2669f4-ca2a-48af-b7f2-010e6b5a1519.png)

Reply:
Changed!